src/main/kotlin/org/jetbrains/bio/io/Bed.kt

```diff
@@ -299,20 +301,25 @@ data class BedFormat(
     }
 }
 
-class BedParser(internal val reader: BufferedReader,
-                private val source: String,
-                val separator: Char
+/**
+ * Reads [BedEntry]-s from the supplied [reader].
+ */
+class BedParser(
+        internal val reader: BufferedReader,
+        private val source: String,
+        val separator: Char,
+        var leniency: Leniency = STRICT
```

Comment:
I'd rather call it `tolerance` of `stringency` to avoid non commonly used terms.

Reply:
I'd say that "leniency" is a commonly used term too, but `stringency` does indeed seem to be used more often.